src/DynamoPackages/PackageManagerExtension.cs

```diff
@@ -233,6 +241,14 @@ private void OnPackageLoaded(Package package)
             // Add new custom nodes to CustomNodePackageDictionary
             foreach (var cn in package.LoadedCustomNodes)
             {
+                if (CustomNodePackageDictionary.ContainsKey(cn.FunctionId))
+                {
+                    OnMessageLogged(LogMessage.Info(
+                        string.Format("{0} contains the custom node {1}, which has already been loaded " +
+                        "by another package. This may cause inconsistent results when determining which " +
+                        "package instances of this custom node are dependent on.", package.Name, cn.Name)
+                        ));
```

Comment:
Same here: do these need to be localized?

Reply:
we have in the past not localized these warnings sent to the console if they are intended for us rather than customers -thats my thinking anyway and there is precedent. What do you think?

Reply:
If that's the case, I think it's okay to leave it as is.